modules/express/package.json

```diff
@@ -73,6 +73,9 @@
     "supertest-as-promised": "https://github.com/BitGo/supertest-as-promised/archive/a7f4b612b9fa090ae33a9616c41862aec2b25c7e.tar.gz",
     "typescript": "^4.2.4"
   },
+  "resolutions": {
```

Comment:
I don't think npm honors the resolutions field, that might be a yarn specific thing. Instead we'd want to use `overrides` 
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides.

if we were going to use resolutions, I think that should go in the root package.json, which _is_ managed by yarn.

Reply:
override is only supported since 8.3.0 right? for node 14 I think it is npm v6. ref: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides

Reply:
We can:
* Put resolution in the root for yarn installers
* Put override in modules/express

In testing with v6.14.17. Looks to resolve correctly.

`build-docker-express` released for customers pull from the root node_modules so that will be correct as well.

Reply:
Pending `ci` tests (for removed `yarn.lock`), we only need to update `express`.